weather.py

```diff
@@ -0,0 +1,220 @@
+import argparse
+import os
+
+months = ['Jan', 'Feb', 'Mar', 'Apr',
+          'May', 'Jun', 'Jul', 'Aug',
+          'Sep', 'Oct', 'Nov', 'Dec']
+
+labels = [
+    'PKT', 'Max TemperatureC', 'Mean TemperatureC',
+    'Min TemperatureC', 'Dew PointC', 'MeanDew PointC',
+    'Min DewpointC', 'Max Humidity', 'Mean Humidity',
+    'Min Humidity', 'Max Sea Level PressurehPa',
+    'Mean Sea Level PressurehPa', 'Min Sea Level PressurehPa',
+    'Max VisibilityKm', 'Mean VisibilityKm', 'Min VisibilitykM',
+    'Max Wind SpeedKm/h', 'Mean Wind SpeedKm/h', 'Max Gust SpeedKm/h',
+    'Precipitationmm', 'CloudCover', 'Events', 'WindDirDegrees'
+]
+
+month_abbr = {
+    1: 'Jan', 2: 'Feb', 3: 'Mar',
+    4: 'Apr', 5: 'May', 6: 'Jun',
+    7: 'Jul', 8: 'Aug', 9: 'Sep',
+    10: 'Oct', 11: 'Nov', 12: 'Dec'
+}
+
+
+def load_weather_data(file_path, year):
+    weather_data = []
+
+    for month_abbr in months:
+        file_name = f"Murree_weather_{year}_{month_abbr}.txt"
+        full_path = os.path.join(file_path, file_name)
+
+        if not os.path.isfile(full_path):
+            continue
+
+        with open(full_path, mode='r') as file:
+            next(file)
+
+            for line in file:
+                line = line.strip()
+
+                values = line.split(',')

```

Comment:
Violation of DRY Principle. We are doing same work in another function for reading.